src/main/io/ledstrip.c

```diff
@@ -92,7 +92,9 @@ static void ledStripDisable(void);
 
 //#define USE_LED_ANIMATION
 
-#define HZ_TO_MICROS(hz) ((int32_t)((1000 * 1000) / (hz)))
+#define HZ_TO_TIMER_DELAY(hz) ((int32_t)((1000 * 1000) / (hz)))
```

Comment:
Good point. I got confused by @ledvinap's comment into thinking the wrong thing was wrong.